src/com/github/Kraken3/AFKPGC/EventHandlers.java

```diff
@@ -46,10 +46,6 @@ public void onPlayerMoveEvent(PlayerMoveEvent event) {
 		registerActivity(event.getPlayer());
 	}	
 	@EventHandler
-	public void onPlayerChatEvent(PlayerChatEvent event) {

```

Comment:
if im understanding this right this would remove chat messages from keeping you from being kicked?